core/io/stream_peer_tcp.cpp

```diff
@@ -65,12 +65,15 @@ void StreamPeerTCP::accept_socket(Ref<NetSocket> p_sock, IP_Address p_host, uint
 
 	peer_host = p_host;
 	peer_port = p_port;
+	_sock->get_address(local_port);
 }
 
-Error StreamPeerTCP::connect_to_host(const IP_Address &p_host, uint16_t p_port) {
+Error StreamPeerTCP::connect_to_host(const IP_Address &p_host, int p_port, int p_local_port) {
```

Comment:
Why did you change the port data type from uint16_t to int? The whole port range fits in a short integer, so we should use it.

Reply:
My choice to use `int` instead of `uint16_t` was to mimic what I saw in other functions, for example:
- `Error PacketPeerUDP::connect_to_host(const IP_Address &p_host, int p_port)`;
- `Error PacketPeerUDP::listen(int p_port, const IP_Address &p_bind_address, int p_recv_buffer_size)`.

I assumed all methods that were _binded_ (accessible from the script side) would use `int` instead of `uint16_t`.